mars/services/task/execution/ray/executor.py

```diff
@@ -249,6 +261,8 @@ async def execute_subtask_graph(
 
         logger.info("Waiting for stage %s complete.", stage_id)
         ray.wait(output_object_refs, fetch_local=False)
+        self._pre_all_stage_progress += self._calculate_cur_stage_progress()
```

Comment:
Here already waited the stage output object refs, can you provide a fast path?

Reply:
We indeed to update `_pre_all_stage_progress ` after all output object refs are finished. It maybe right here.

Reply:
We can just use `.self._cur_stage_tile_progress` as current stage progress instead of calculation.